about.css

```diff
@@ -0,0 +1,417 @@
+}
+
+.description {
+  color: #141313;
+  background-color: #fff;
+  line-height: 1.5rem;
+  font-size: 1rem;
+  font-weight: 400;
+  padding: 1rem;
+}
+
+.details-presentation { padding: 2rem 0%; }
+
+.email {
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+.info {
+  color: #838080;
+  font-size: 1rem;
+  font-weight: 600;
+  line-height: 1.4rem;
+  padding: 1rem;
+}
+
+.section-title {
+  font-size: 1.6rem;
+  text-align: center;
+}
+
+.logo-section .container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.partners img {
+  width: 20%;
+}
+
+.flex-container div {
+  width: 40%;
+}
+
+.flex-container div img {
+  width: 100%;
+}
+
+.event p { color: #fff; }
+
+.logo-section .container img {
+  width: 70%;
+}
+
+.past-events-sections .container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  margin-top: 2rem;
+  margin-bottom: 3rem;
+}
+
+.events {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.bkg-event-picture {
+  width: 90%;
+  transition: scale 0.4s ease-in-out;
+  transition: all 1s ease-in;
+  background-image: url(images/pexels-on-restaurant.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.event {
+  color: #fff;
+  background-color: #e7350d7a;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 3rem 5rem;
+}
+
+.bkg-event-picture:hover {
+  transform: scale(1.2) rotate(360deg);
+}
+
+.partner-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #3e3e3e;
+  gap: 1rem;
+  padding: 2rem;
+}
+
+.partner-section h2 {
+  color: #838080;
+}
+
+.partners {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  width: 100%;
+}
+
+.flex-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2rem;
+}
+
+footer {
+  width: 100%;
+}
+
+.footer-text {
+  padding: 0%;
+  width: 50%;
+  margin: 0%;
+}
+
+.footer-text p {
+  font-size: 0.8rem;
+}
+
+.desktop {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .container {
+    max-width: 750px;
+    margin: auto;
+    padding: 0%;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  h2 {
+    padding: 1rem;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .nav-menu {
+    display: flex;
+  }
+
+  li a { text-decoration: none; }
+
+  .logo-section .container img {
+    width: 50%;
+  }
+
+  .events {
+    flex-direction: row;
+    gap: 1rem;
+  }
+
+  .bkg-event-picture { width: 45%; }
+
+  footer {
+    padding: 2rem;
+    color: #fff;
+    font-size: 0.8rem;
+    background-color: #2e2e2e;
+  }
+
+  .desktop {
+    display: flex;
+  }
```

Comment:
- [ ] Kindly avoid code repetition in accordance to DRY principle, you've done the same in line 387. 👍

Reply:
I would like to tell that this code is useful in my code to avoid things to apear in desktop and mobile version according to their porpouse in the html.